tests/gis_tests/test_geoforms.py

```diff
@@ -401,3 +401,11 @@ class PointForm(forms.Form):
         form = PointForm(data={'p': point.json})
         self.assertTrue(form.is_valid())
         self.assertEqual(form.cleaned_data['p'].srid, 4326)
+
+    def test_base_geometry_widget(self):
+        """
+        Regression test for issue #28105
+        """
```

Comment:
You can remove the ticket mention, we try to only use them when the issue really needs context. You might want to change the name of the test to be more descriptive of the issue. Something along the lines of `test_base_geometry_attrs_collision`?